js/app.js

```diff
@@ -358,7 +358,7 @@ $(document).ready(function () {
           else {
             $(el).parent().attr('action', 'update');
           }
-          $(el).parent().addClass('uncommitted');
+          // $(el).closest('label-entry').addClass('uncommitted');
```

Comment:
Suggestion to this section:
```
if ($(el).val() === $(el).attr('data-orig-val')) {
          $(el).closest(".label-entry").attr('action', 'none');
          $(el).closest(".label-entry").removeClass('uncommitted');
        }
        else {
          if ($(el).closest(".label-entry").attr('new') === 'true') {
            $(el).closest(".label-entry").attr('action', 'create');
          }
          else {
            $(el).closest(".label-entry").attr('action', 'update');
          }
          $(el).closest(".label-entry").addClass('uncommitted');
        }
```

Reply:
As I mentioned in the issue ticket, ***a portion of*** of the code within the function starting on line `337` seems to be redundant. Commenting out lines `350` to `363` has no effect on functionality as far as I can tell, while also addressing your follow-up issue.

Lines `278`-`299` should be sufficient for monitoring all label input changes, and we should be able to delete lines `350` to `363`, or comment out for the time being should any bugs arise in the future.

Please feel free to test the most recently pushed branch, and let me know if you catch any bugs.

Reply:
I agree